nemo_text_processing/g2p/data/data_utils.py

```diff
@@ -121,41 +130,76 @@ def english_text_preprocessing(text, lower=True):
     return text
 
 
-def _word_tokenize(words):
+def _word_tokenize(words: List[Tuple[str, str, str]]) -> List[Tuple[List[str], bool]]:
     """
-    Convert text (str) to List[Tuple[Union[str, List[str]], bool]] where every tuple denotes word representation and
-    flag whether to leave unchanged or not.
-    Word can be one of: valid english word, any substring starts from | to | (unchangeable word) or punctuation marks.
-    This function expects that unchangeable word is carefully divided by spaces (e.g. HH AH L OW).
-    Unchangeable word will be splitted by space and represented as List[str], other cases are represented as str.
+    Process a list of words and attach indicators showing if each word is unchangeable or not. Each word representation
+    can be one of valid english word, any substring starting from | to | (unchangeable word), or punctuation marks. This
```

Comment:
`can be one of valid english word` is this intended for Eng only?

Reply:
good catch! now this func is not English-specific. It should be `one of valid Latin word`.